packages/client/src/__tests__/binaryEngine.test.ts

```diff
@@ -1,6 +1,6 @@
 import { BinaryEngine } from '@prisma/engine-core'
+import { ClientEngineType, getClientEngineType } from '@prisma/sdk'
 import path from 'path'
-import { ClientEngineType, getClientEngineType } from '../runtime/utils/getClientEngineType'
 
```

Comment:
What prompted the import changes? Just curious

Reply:
I needed to use `getClientEngineType` in CLI, and importing it like

```
import { getClientEngineType } from '@prisma/client/src/runtime/utils/getClientEngineType'
```

didn't feel right (even though it technically worked).

Reply:
Trying to think if there was a reason it was like this 🤔

Reply:
It wasn't like that, this file was only used inside `packages/client` via relative imports. My usage was the first one outside of `@prisma/client`, so I had to move it to sdk (or do the hacky import directly from `@prisma/client` that only worked at run time because we bundle things).